weetweet.py

```diff
@@ -1051,28 +1133,39 @@ def hook_commands_and_completions():
-        "you can check out your limits with the rate_limits/limits command.\n"
-        "_Most_ commands in this plugin only uses one call. If you want to check old tweets\n"
-        "in your home timeline it's better to request many tweets in one go.\n"
-        "That way you don't have to request new tweets as often to go further back in the timeline.\n"
-        "And thus you are less likely to hit the limit of requests you can do in the 15 min time window.\n"
-        "\nYou can write newlines in your tweet with html newline '&#13;&#10;' (you can autocomplete it)\n"
-        "\nThe 'number' next to the nicks in the chat window is the <id> of the tweet it's used\n"
-        "in the some of the twitter plugin commands.\n\n"
-        "Command desc:\n"+ "\n".join(desc_list),
-        " || ".join(compl_list),
-        "my_command_cb", "")
+                         " | ".join(com_list),
+                         "You can type all of these command in the twitter buffer if you add a\
+                                 ':' before the command, IE:\n"
+                         ":limits\n\n"
+                         "If you don't type a command in the twitter buffer you will tweet that\
+                                 instead, \n"
+                         "text after 280 chars will turn red to let you know were twitter will\
+                                 cut off your tweet.\n\n"
+                         + weechat.color("*red") + "NOTE:\n"
+                         "There are limits on how many twitter api calls you can do, some calls\
+                                 are _quite_ restricted.\n"
+                         "So if you get HTML errors from the twitter lib you probably exceeded\
+                                 the limit\n"
+                         "you can check out your limits with the rate_limits/limits command.\n"
+                         "_Most_ commands in this plugin only uses one call. If you want to check\
+                                 old tweets\n"
+                         "in your home timeline it's better to request many tweets in one go.\n"
+                         "That way you don't have to request new tweets as often to go further\
+                                 back in the timeline.\n"
+                         "And thus you are less likely to hit the limit of requests you can do\
+                                 in the 15 min time window.\n"
+                         "\nYou can write newlines in your tweet with html newline '&#13;&#10;'\
+                                 (you can autocomplete it)\n"
+                         "\nThe 'number' next to the nicks in the chat window is the <id> of the\
+                                 tweet it's used\n"
+                         "in the some of the twitter plugin commands.\n\n"
+                         "Command desc:\n"+ "\n".join(desc_list),
+                         " || ".join(compl_list),
+                         "my_command_cb", "")
```

Comment:
I think this became a lot harder to read now that the intendation jumps in a every newline